f2/src/forms/MidFeedbackForm.js

```diff
@@ -60,8 +61,9 @@ export const MidFeedbackForm = props => {
           </InfoCard>
         </Row>
       ) : (
-        <Box as="details">
+        <Box>
           <Button
```

Comment:
The button is rendered `as="summary"` rendering it inaccessible with keyboard navigation.  Just removing this prop should fix this i believe. I also want to see how voice over reacts to clicking on the button. See #1551 for similar solution!